src/make_watertight/Arc.hpp

```diff
@@ -73,7 +70,7 @@ class Arc {
 
   /// goes through curve_sets and finds any curves with coincident ( dist. apart <= FACET_TOL) front and back points.
```

Comment:
```suggestion
  /// goes through curve_sets and finds any curves with coincident ( dist. apart <= facet_tol) front and back points.
```

Reply:
Ah here it is, the source of the `FACET_TOL` --> `facet_tol` comments. I'm going to revert this change for now so they can be dealt with all together at another time.